projects/ngx-indexed-db/src/lib/ngx-indexed-db.service.ts

```diff
@@ -305,13 +305,12 @@ export class NgxIndexedDBService {
           const objectStore = transaction.objectStore(storeName);
 
           transaction.oncomplete = () => {
-            const request = objectStore.get(key) as IDBRequest<T>;
-            request.onsuccess = (event: Event) => {
-              obs.next((event.target as IDBRequest<T>).result);
-            };
-            request.onerror = (event: Event) => {
-              obs.error(event);
-            };
```

Comment:
What about error handling?

Reply:
Error handling is present in the getByKey method. Other methods also have this management.

Reply:
Hi @assuncaocharles,
any news?